OrbitGl/CaptureWindow.cpp

```diff
@@ -515,6 +515,34 @@ void CaptureWindow::Draw() {
     // Vertical green line at mouse x position
     ui_batcher_.AddVerticalLine(pos, -world_height_, kZValueText, Color(0, 255, 0, 127));
   }
+  DrawScreenSpace();
+
+  // We start by getting all layers
+  std::vector<float> all_layers = time_graph_.GetBatcher().GetLayers();
+  auto ui_layers = ui_batcher_.GetLayers();
+  all_layers.insert(all_layers.end(), ui_layers.begin(), ui_layers.end());
+  auto text_ui_layers = text_renderer_.GetLayers();
+  all_layers.insert(all_layers.end(), text_ui_layers.begin(), text_ui_layers.end());
+  auto text_tg_layers = time_graph_.GetTextRenderer()->GetLayers();
+  all_layers.insert(all_layers.end(), text_tg_layers.begin(), text_tg_layers.end());
```

Comment:
nit. could be replaced by: 
Append(all_layers, ui_batcher_.GetLayers());
Append(all_layers, text_renderer_.GetLayers());
Append(all_layers, time_graph_.GetTextRenderer()->GetLayers());

It's our own util method that wraps the not-so-eye-pleasing std api. I find it much more readable.

Reply:
Yes, clearly!